packages/shorebird_cli/lib/src/commands/build/build_apk_command.dart

```diff
@@ -68,8 +69,10 @@ class BuildApkCommand extends ShorebirdCommand
     buildProgress.complete();
 
     final apkPath = flavor != null
-        ? './build/app/outputs/apk/$flavor/release/app-$flavor-release.apk'
-        : './build/app/outputs/apk/release/app-release.apk';
+        ? p.normalize(
+            './build/app/outputs/apk/$flavor/release/app-$flavor-release.apk',
+          )
+        : p.normalize('./build/app/outputs/apk/release/app-release.apk');
```

Comment:
TIL about normalize.

Does `normalize` handle different path separators? Based on the docs, it looks like it just removes nested `..`s, etc.

Reply:
Ah good catch...didn't want to use canonicalize since it returns absolute paths :/

Reply:
Adjusted to use `join`